openfl/interface/aggregator.py

```diff
@@ -91,6 +92,25 @@ def start_(plan, authorized_cols, task_group):
         parsed_plan.config["assigner"]["settings"]["selected_task_group"] = task_group
         logger.info(f"Setting aggregator to assign: {task_group} task_group")
 
+    # Check if remote attestation is enabled in the plan configuration
+    if parsed_plan.config["network"]["settings"].get("enable_remote_attestation", False):
+        # Fetch remote attestation environment variables
+        attestation_env = attestation_utils.fetch_attestation_env_vars()
+        if attestation_env is not None:
+            attestation_mr = attestation_utils.AttestationManager(
```

Comment:
`mr` may be confused with "measurement", so I suggest using the full `attestation_manager` name.

Reply:
done